ci-operator/jobs/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel-periodics.yaml

```diff
@@ -0,0 +1,75 @@
+periodics:
```

Comment:
Isn't this all generated by make jobs/make update?

Reply:
It is generated, yes, so you don't need to review. However, the code should be committed.